### PR TITLE
WP-5569: preserve column order of data imported

### PR DESCRIPTION
**Pull request by tomwang-varicent**
jira: https://varicent.atlassian.net/browse/WP-5569
app pr: https://github.com/symon-ai/app/pull/3545/

v0.3.2 should be created.

### CODE REVIEW

File: tap_oracle/__init__.py [hunk 1]
```diff
@@ -552,8 +562,23 @@ def main_impl():
...
+            order = stream['column_order']
+
+            for column in order:
+               new_properties[column] = old_properties[column]
+
+            stream['schema']['properties'] = new_properties
+
+      args.catalog = Catalog.from_dict(args.properties)
+      
       do_sync(conn_config, args.catalog, args.config.get('default_replication_method'), state)
```
Comment (ali-varicent) [inline tap_oracle/__init__.py:582]: >      do_sync(conn_config, args.catalog, args.config.get('default_replication_method'), state)
should this change to args.properties ? would the rest work fine ?

**Review by ali-varicent — CHANGES_REQUESTED** [review]
update the readme as well with the new command

File: tap_oracle/__init__.py (hunk 1 above)
Comment (tomwang-varicent) [reply tap_oracle/__init__.py:582]: `args.catalog = Catalog.from_dict(args.properties)`
It convert args.properties, which was read from the json file as a dictionary, to args.catalog, which is a Catalog instance needed by the rest.

Comment (ali-varicent) [reply tap_oracle/__init__.py:582]: @tomwang-varicent thanks for clarifying.
So plz just update all the readme files for the taps you changed to have the new commands

**Review by ali-varicent — APPROVED** [review]

**Pull request merged**